monitoring/ceph-mixin/grafana-json-to-libsonnet.py

```diff
@@ -0,0 +1,23 @@
+import json, sys
+from jinja2 import Environment, FileSystemLoader
+
+if (len(sys.argv[1:]) < 2):
+    raise Exception(
+        "Need to pass two parameters, like: python3 grafana-libsonnet-to-json.py <input_json_path> <output_libsonnet_path>"
+    )
+
+json_path = sys.argv[1] # example: dashboards_out/new.json
+output_path = sys.argv[2] # example: dashboards/new.libsonnet
```

Comment:
optionally, if you don't give an output path, can it assume from the file itself. The idea is to generate libsonnet in the end, so we may don't even need to ask the person who run the script to pass something like `new.libsonnet`. If the json name is `new.json`, then the script will create `new.libsonnet`.